security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -364,7 +370,8 @@ public void handleUrlParamRequest(HttpServletRequest request, HttpServletRespons
      * sometimes used by the underlying webapps (e.g. mapfishapp and the mfprint
      * configuration). hence we need to allow it in the following "params" array.*
      */
-    @RequestMapping(value = "/**", params = { "!login" })
+    @RequestMapping(value = "/**", params = { "!login" }, method = { GET, POST, HEAD, OPTIONS, PUT, PATCH, DELETE,
+            TRACE })
```

Comment:
I still cannot get why it is necessary to pass explicitly the methods that should match, it should already be the default ones:
https://docs.spring.io/spring/docs/4.3.22.RELEASE/javadoc-api/org/springframework/web/bind/annotation/RequestMapping.html#method--

But I do trust you anyway, if it's necessary then we should go for it

Reply:
I don't understand it neither... not enough spring guru to find the actual interface implementation